community/kernel/src/main/java/org/neo4j/kernel/api/DataRead.java

```diff
@@ -80,7 +79,7 @@ PrimitiveLongIterator nodesGetFromIndexScan( IndexDescriptor index )
      * Returns node id of unique node found in the given unique index for value or
      * {@link StatementConstants#NO_SUCH_NODE} if the index does not contain a
      * matching node.
-     *
+     * <p/>
```

Comment:
That's an intellij auto-formatter issue. If there's a blank line in the javadoc, it "helpfully" puts `<p/>` in there.